README.md

```diff
@@ -29,7 +29,7 @@ This project is structered as a [foundry](https://getfoundry.sh/) project with t
 |:---:|:---:|:---:|
 | `ICS26Router.sol` | IBC Eureka router handles sequencing, replay protection, and timeout checks. Passes proofs to `ICS02Client.sol` for verification, and resolves `portId` for app callbacks. Provable IBC storage is stored in this contract.  | ✅ |
 | `ICS02Client.sol` | IBC Eureka light client router resolves `clientId` for proof verification. It also stores the counterparty information for each client. | ✅ |
-| `SdkICS20Transfer.sol` | IBC Eureka transfer application to send and receive tokens to/from `CosmosSDK`. | ✅ |
+| `ICS20Transfer.sol` | IBC Eureka transfer application to send and receive tokens to/from `CosmosSDK`. | ✅ |
```

Comment:
I suppose the description could be updated here as well, as it would support send/receive to/from any IBC chain now.

Reply:
Good point. I adjusted it now.